cypress/e2e/awx/administration/management-jobs.cy.ts

```diff
@@ -18,128 +40,107 @@ describe('Management Jobs', () => {
       cy.wait('@getManagementJobsListPage')
         .its('response.body.results')
         .then((results: SystemJobTemplate[]) => {
-          const responseJobsList = results?.map((job: SystemJobTemplate) => job.name);
-          expect(responseJobsList).to.have.members(managementJobsList);
-          cy.get('.pf-v5-c-table__tbody tr').should('have.length', responseJobsList.length);
-          cy.get('.pf-v5-c-table__tbody tr').each(($row, index) => {
-            cy.wrap($row).find('td').eq(0).should('contain', responseJobsList[index]);
-          });
+          const jobId = results.find((job) => job.name === jobName)?.id;
+          if (jobId) {
+            cy.intercept('POST', `/api/v2/system_job_templates/${jobId}/launch/`).as('launchJob');
+            cy.clickTableRowAction('name', jobName, 'launch-management-job', {
+              inKebab: false,
+              disableFilter: true,
+            });
+            cy.wait('@launchJob')
+              .its('response.body.id')
+              .then((jobId: string) => {
+                cy.verifyPageTitle(jobName);
+                cy.url().should('include', `/jobs/management/${jobId}/output`);
+                cy.waitForManagementJobStatus(jobId).then((response: SystemJobTemplate) => {
+                  const status = response.status;
+                  let resultStatus;
+                  if (status === 'successful') {
+                    resultStatus = 'Success';
+                  }
+                  cy.contains('a[role="tab"]', 'Details').click();
+                  cy.get('[data-cy="id"]').should('have.text', jobId).should('be.visible');
+                  cy.get('[data-cy="name"]').should('have.text', jobName).should('be.visible');
+                  cy.get('[data-cy="status"]')
+                    .should('have.text', resultStatus)
```

Comment:
This line is failing for me when testing locally, maybe an issue with the command `waitForManagementJobStatus`. Will need to investigate more
<img width="1917" alt="Screenshot 2024-05-07 at 5 46 00 PM" src="https://github.com/ansible/ansible-ui/assets/112966709/906d06f5-a4da-44a7-bd39-46c931a1d0c4">

Reply:
since the status is asserted from the custom command, I'm considering removing this line. @prat98

Reply:
because if the jobs fails the it won't be Success in the UI

Reply:
That sounds good to me